src/App.tsx

```diff
@@ -5,29 +5,61 @@ import AddItemForm from './components/AddItemForm';
 import TaskDashboard from './components/TaskDashboard';
 import TaskItem from './components/TaskItem';
 import { TaskItemType } from './types/TaskItemType';
+import EditTaskModal from './components/EditTaskModal';
 
 function App() {
   const [taskList, setTaskList] = useState<TaskItemType[]>([]);
 
-  const handleAddTask = (taskTitle: string) => {
+  const [isEditModalOpen, setIsEditModalOpen] = useState<boolean>(false);
```

Comment:
```suggestion
  const [showEditModal, setShowEditModal] = useState<boolean>(false);
```
Fala mano, talvez dê para simplificar esse hook, segue uma sugestão acima.